packages/library-legacy/scripts/typegen.sh

```diff
@@ -6,20 +6,5 @@ pnpm tsc -p tsconfig.d.json -d
 # Flatten typescript declarations
 pnpm rollup -c rollup.config.types.mjs
 
-# Replace export with closing brace for module declaration
-sed -i.bak '$s/export {.*};/}/' lib/index.d.ts
-
-# Replace declare's with export's
-sed -i.bak 's/declare/export/g' lib/index.d.ts
-
-# Prepend declare module line
-sed -i.bak '2s;^;declare module "@solana/web3.js" {\n;' lib/index.d.ts
```

Comment:
This doesn't seem to work anymore.

https://github.com/solana-labs/solana-web3.js/actions/runs/9665989204/job/26664439889